patches/0010-LPS-139565-When-upgrading-from-6.2-to-7.1-image-widt.patch

```diff
@@ -0,0 +1,53 @@
+From 041396d0d9997e8c078eb17a50f7348a4d2a64e8 Mon Sep 17 00:00:00 2001
+From: Minhchau <minhchau.dang@liferay.com>
+Date: Tue, 28 Sep 2021 11:18:40 -0700
+Subject: [PATCH] LPS-139565 When upgrading from 6.2 to 7.1, image width/height
+ in web content is lost
+
+Move inline styles for height/width to image attributes
+---
+ plugins/image2/dialogs/image2.js | 16 ++++++++++++++++
+ plugins/image2/plugin.js         |  3 +++
+ 2 files changed, 19 insertions(+)
+
+diff --git a/plugins/image2/dialogs/image2.js b/plugins/image2/dialogs/image2.js
+index b09a05f15..0782fa2e6 100644
+--- a/plugins/image2/dialogs/image2.js
++++ b/plugins/image2/dialogs/image2.js
+@@ -394,6 +394,22 @@ CKEDITOR.dialog.add( 'image2', function( editor ) {
+ 			// Create a "global" reference to widget's image.
+ 			image = widget.parts.image;
+ 
++			var styleHeight = image.$.style.removeProperty('height');
++
++			if (styleHeight) {
++				styleHeight = styleHeight.replace('px', '');
++				widget.data.height = styleHeight;
++				image.$.setAttribute('height', styleHeight);
++			}
++
++			var styleWidth = image.$.style.removeProperty('width');
++
++			if (styleWidth) {
++				styleWidth = styleWidth.replace('px', '');
++				widget.data.width = styleWidth;
++				image.$.setAttribute('width', styleWidth);
++			}
++
+ 			// Reset global variables.
+ 			srcChanged = userDefinedLock = lockRatio = false;
+ 
+diff --git a/plugins/image2/plugin.js b/plugins/image2/plugin.js
+index e85d58c2b..59423cb79 100644
+--- a/plugins/image2/plugin.js
++++ b/plugins/image2/plugin.js
+@@ -1190,6 +1190,9 @@
+ 				nativeEvt, newWidth, newHeight, updateData,
+ 				moveDiffX, moveDiffY, moveRatio;
+ 
++			image.$.style.removeProperty('height');
++			image.$.style.removeProperty('width');
```

Comment:
Nice catch that we need to fix resize! :+1: 

The only issue here is there is a brief flash of image in its original size, before resize sets width and height attributes. I tried making a gif to show this, but the flash happens too quickly to be captured :slightly_smiling_face:   

To fix this, we just need to set width and height attributes here, before we remove the properties.

Reply:
@markocikos I've applied the requested changes.